papa/src/main/java/papa/InteractionRuleClient.kt

```diff
@@ -119,18 +120,16 @@ private class InteractionEngine<ParentEventType : Any>(
      * called with an unknown [Runnable].
      */
     private val cancelOnTimeout: Runnable = Runnable {
-      SafeTrace.logSection {
-        "PAPA-cancel:timeout"
-      }
+      Trace.beginSection("PAPA-cancel:timeout")
+      Trace.endSection()
```

Comment:
why not use the lambda version with `{}` ?

or at least bring back `logSection` as helper function here?